airflow/utils/logging.py

```diff
@@ -80,8 +81,7 @@ def read(self, remote_log_location, return_error=False):
         logging.error(err)
         return err if return_error else ''
 
-
-    def write(self, log, remote_log_location, append=False):
+    def write(self, log, remote_log_location, append=True):
```

Comment:
YAY this means we'll now have the full set of logs across retries rather than just seeing the most recent!